tests/AutoInstrumentation/scenarios/composer_with_ddtrace_dependency/composer.json

```diff
@@ -1,5 +1,8 @@
 {
     "require": {
         "datadog/dd-trace": "0.49.0"
+    },
+    "config": {
+        "disable-tls": true
```

Comment:
Is there a reason these weren't committed before but are now? IIRC some composer command would run and disable it, but it wasn't committed. Or am I mis-remembering?

Reply:
Looks like it, will remove from PR.